cmake/modules/FindIso9660pp.cmake

```diff
@@ -31,12 +33,12 @@ set(ISO9660PP_VERSION ${PC_ISO9660PP_VERSION})
 
 include(FindPackageHandleStandardArgs)
 find_package_handle_standard_args(Iso9660pp
-                                  REQUIRED_VARS ISO9660PP_LIBRARY ISO9660PP_INCLUDE_DIR ISO9660_LIBRARY ISO9660_INCLUDE_DIR
+                                  REQUIRED_VARS ISO9660PP_LIBRARY ISO9660PP_INCLUDE_DIR CDIOPP_INCLUDE_DIR ISO9660_LIBRARY ISO9660_INCLUDE_DIR CDIO_LIBRARY CDIO_INCLUDE_DIR
```

Comment:
not needed

Reply:
Required to be here.

`cdio++` is needed if and only if `iso9600++` is used.

Reply:
Removed